src/print.c

```diff
@@ -53,6 +56,21 @@ int apk_get_screen_width(void)
 	return apk_screen_width;
 }
 
+const char *apk_get_human_size(off_t size, off_t *dest)
+{
+	size_t i;
+	off_t s;
+
+	assert(size >= 0);
```

Comment:
Just wondering: Should we make this an unconditional run-time check? Would it be possible for someone to create a `.PKGINFO` with a negative package size? Because if so: This would probably trigger the assert, wouldn't it?

Reply:
It's converted to `size_t` which is unsigned. So this is ok, though the assert can bite later.

Reply:
If in doubt we could just replace the `assert(3)` invocation with:

```C
if (size < 0)
  size = (off_t)llabs(size);
```

For `size = 0` the code should actually work correctly.

Reply:
Mmm... I'd probably keep the sign on the return value then. But that's debatable. I think eitherway is good enough.